UPGRADE-1.7.md

```diff
@@ -0,0 +1,12 @@
+# UPGRADE FROM `v1.6.*` TO `v1.7.0`
+
+* **BC BREAK**: Edit your fos_rest config to enable the body listener as following:
+```yaml
+fos_rest:
+    # [...]
+    body_listener:
+        enabled: true 
+```
+
+* **BC BREAK**: `ViewHandler` constructor now requires `FOS\RestBundle\View\ConfigurableViewHandlerInterface` as first argument instead of 
```

Comment:
```suggestion
* **BC BREAK**: `Sylius\Bundle\ResourceBundle\Controller\ViewHandler` constructor now requires `FOS\RestBundle\View\ConfigurableViewHandlerInterface` as first argument instead of 
```